src/engine/SCons/Tool/docbook/docbook-xsl-1.76.1/extensions/docbook.py

```diff
@@ -34,9 +34,9 @@ def adjustColumnWidths(ctx, nodeset):
     # Get the nominal table width
     varString = lookupVariable(tctxt, "nominal.table.width", None)
     if varString is None:
-        nominalWidth = 6 * pixelsPerInch;
+        nominalWidth = 6 * pixelsPerInch
```

Comment:
Is this SCons logic, or just copied in from docbook package?

Reply:
docbook.  git, and thus pycharm, doesn't know the difference (I've commented on that before - the imported docbook stuff probably ought to be a submodule).  In any case a trailing semicolon isn't needed and it's not like we haven't changed docbook code (from upstream, doesn't come pyt3-clean)